### DISCUSSION TITLE
Text explaining the "Attach volume" function

### DISCUSSION THREAD

**Discussion by merlinbest5** [biigle/core#857] [General]
I was excited to see the "Attach volumes" function (or possibly it's been there all along and I never noticed until now!), but I experienced some confusion and thought it would help to a) check my understanding and b) if correct, tweak the text in the manual to clarify. When users "attach" a volume, it seems to create a copy/clone rather than some kind of linked version (as in, changes to the new copy don't appear in the original). Here's the text from the manual:

"Project admins can create or delete volumes. To create a volume, click the [Create Volume] button of the volumes tab in the project overview. A new volume will be automatically attached to the project it was created for. Volumes can be shared between projects, too. You can attach existing volumes of any project where you are also admin. To do this, enter the name of the volume you want to attach in the input field on the top right of the volumes tab and hit enter. Hover the mouse over a volume thumbnail and click the [garbage can icon] button to detach or delete a volume. A volume is deleted only if this is the last project that it is attached to (there will be an additional request for confirmation in this case). Be careful when doing this since deleting a volume deletes all annotations and cannot be undone!"

I went ahead and rewrote the section according to my understanding, just in case it's helpful. Wasn't sure how to include the icons: "Project admins can create or delete volumes. To create a volume, click the Create Volume button of the volumes tab in the project overview. A new volume will be automatically attached to the project it was created for. Alternatively, you can attach a copy of a volume from an existing project if you are also an admin. To do this, enter the name of the volume you want to attach in the [Attach volumes] input field on the top right of the volumes tab and hit enter. Note that this creates a copy and is not linked to the original volume - changes made to the new copy will not be reflected in the original. Hover the mouse over a volume thumbnail and click the [garbage can icon] button to detach or delete a volume. A volume is deleted only if this is the last project that it is attached to (there will be an additional request for confirmation in this case). Be careful when doing this since deleting a volume deletes all annotations and cannot be undone!"

**Comment by dlangenk** [comment]
It's been around since a few years already ;-) Actually it should create a linked version, i.e.  if you change the original or the attached volume both will have the same changes (Label trees are not automatically linked, so in Largo for example some things will not be displayed as they cannot be selected because the label tree might not be present). Instead of linking you can also clone a volume (hover over a volume and press the small copy icon), which creates a real individual copy. 
However if there is an error please let us know.
I will have a look at the text you wrote on monday. Thanks a lot :-)

P.S. The video sound can now be toggled on and off in the settings since 3 hours, cmp. #819

**Reply by merlinbest5** [reply]
Thank you @dlangenk. I tested it out before creating this discussion but my internet connection must've been disrupted as it didn't show the change. I see how they are attached now :) Ignore my tweaks to the text, unless you want to add something to clarify this to others, in which case you're welcome to use some version of the section below:

"Project admins can create or delete volumes. To create a volume, click the [Create Volume] button of the volumes tab in the project overview. A new volume will be automatically attached to the project it was created for. Alternatively, you can attach existing volumes of any project if you are also an admin. To do this, enter the name of the volume you want to attach in the [Attach volumes] input field on the top right of the volumes tab and hit enter. Changes in either copy will be reflected in both volumes. Hover the mouse over a volume thumbnail and click the [garbage can icon] button to detach or delete a volume. A volume is deleted only if this is the last project that it is attached to (there will be an additional request for confirmation in this case). Be careful when doing this since deleting a volume deletes all annotations and cannot be undone!"

**Reply by mzur** [reply]
Thanks for the suggestion. I found the "changes in either copy..." part a bit misleading because the volume is not actually copied. Both projects show the exact same volume. So here is what I ended up with (diff from the original text):

![image](https://github.com/biigle/core/assets/2457311/fab72cd5-40ee-4891-b1ed-e939b4154775)

I've opened https://github.com/biigle/core/issues/859 for that.
